#status of different systems, to display on the GUI
string jaco_status
string camera_status
string main_status
string moving_status